stopcovid/clients/sqs.py

```diff
@@ -27,7 +28,7 @@ def publish_outbound_sms_messages(outbound_sms_messages: List[OutboundSMS]):
 
     entries = [
         {
-            "Id": phone,
+            "Id": re.sub("[^0-9]", "", phone),
```

Comment:
stripping out +-signs too? (what about international?)

Reply:
These ids are only used in the enqueue side. You're supposed to use them by looking at the response SQS gives you to be able to see which entries were successful and which werent successful. This is pretty misleading though. If they dont matter we should just use index or uuids

Reply:
Will fix in a subsequent pr